src/javascripts/helpers/data/chaosData.js

```diff
@@ -0,0 +1,52 @@
+// import equipmentData from './equipmentData';
+import axios from 'axios';
+
+const randomEquipment = (category) => new Promise((resolve, reject) => {
+  axios
+    .get(`https://nutshell-part-two.firebaseio.com/${category}.json`)
+    .then((response) => {
+      const arrayLength = Object.values(response.data).length;
+      const randomNumber = Math.floor(Math.random() * arrayLength);
+      const choatic = Object.values(response.data)[randomNumber];
+      resolve(choatic.name);
+    })
+    .catch((error) => reject(error));
+
+  // equipmentData
```

Comment:
Let's not put commented code in prod. Go ahead and remove this please.

Reply:
It's been removed